frontend/interfaces/host.ts

```diff
@@ -83,6 +85,18 @@ export interface IPackStats {
   type: string;
 }
 
+export interface IHostPolicyQuery {
+  id: number;
+  hostname: string;
+  status: string | undefined;
```

Comment:
can we change this to `status?: string;`?

Reply:
Good idea!